### PR TITLE
5 reasons why the webbing PR has enraged me like no other

### PR DESCRIPTION
**Pull request by Bawhoppen**
1: Was merged despite the failure to come to a popular conclusion
2: Was merged without WJ's approval
3: Looks ungodly awful
4: Was not properly done as a subtype and was just half-assed
5: My revert PR was closed within 24 hours

### CODE REVIEW

**Comment by Bawhoppen** [comment]
this PR adds it as a proper subtype atleast by the way

**Comment by FantasticFwoosh** [comment]
This aint nothing precious kid.

**Comment by Bawhoppen** [comment]
[10:47:59] <Bawhoppen> fwoosh I know you're going to comment
[10:48:00] <Bawhoppen> but don't

**Comment by WJohn** [comment]
I mean at least it's a subtype now so w/e.

**Comment by Iamgoofball** [comment]
1: This isn't a democracy.
2: No news is good news.
3: Sounds like a personal opinion.
4: It was supposed to replace the belt.
5: We have a rule against kneejerk reverts in CONTRIBUTING.MD.

**Comment by Bawhoppen** [comment]
then why has my stuff been knee-jerk reverted and speedmerged?
checkmate

**Comment by Iamgoofball** [comment]
Provide proof.

**Comment by Bawhoppen** [comment]
https://github.com/tgstation/tgstation/pull/25236
https://github.com/tgstation/tgstation/pull/25620

**Comment by fludd12** [comment]
>March 18th

>March 29th

**Comment by Iamgoofball** [comment]
@KorPhaeron @optimumtact if we're not going to enforce it why have the rule

**Comment by Iamgoofball** [comment]
@fludd12 
> Razharas merged commit 86cbbaf into tgstation:master on Mar 29
1 check passed

**Comment by Bawhoppen** [comment]
@fludd12 look when it got merged

**Comment by KorPhaeron** [comment]
How is it halfassed to replace what would be an unused item you meme

**Comment by Bawhoppen** [comment]
what do you mean unused?

**Comment by KorPhaeron** [comment]
The red belt is now unused since I replaced it with the rig, so why bother keeping it in the code?

**Comment by Bawhoppen** [comment]
For cases where it could be used in other contexts? I have it in some space ruins and all that
Also this does change the traitors to get the belt back

**Comment by optimumtact** [comment]
Well if you'd like baw I'll keep this one open for 24 hours before closing it

**Comment by Bawhoppen** [comment]
why would you even close this one? you guys won, this is just making a subtype so we can have both

**Comment by kevinz000** [comment]
@KorPhaeron why do you do this? character sprites look horribad without belts ;~~~~;

**Comment by Bawhoppen** [comment]
6. the belt is also a really classic thing that removal detracts from good feeling

**Comment by optimumtact** [comment]
Because you don't deserve both

**Comment by Bawhoppen** [comment]
7. oranges need helps

**Comment by Pascal125** [comment]
I think i deserve both.

**Comment by optimumtact** [comment]
You have a furry avatar

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -914,8 +914,8 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	cost = 3
 
 /datum/uplink_item/device_tools/military_belt
-	name = "Chest Rig"
-	desc = "A robust seven-slot set of webbing that is capable of holding all manner of tatical equipment."
+	name = "Military Belt"
```
Comment (RemieRichards) [inline code/modules/uplink/uplink_item.dm]: Now you can't buy the rig, allow it to be bought.

Comment (Bawhoppen) [inline code/modules/uplink/uplink_item.dm]: yes I said that I am aware that this gives the belt back to traitors so ops can have their webbing

**Comment by jughu** [comment]
Theres two sides of the story

People that are fucking attached to recolored belts for jobs and a very ugly red one for taitortots

And a tactical ass looking rig for TACTICAL operations which doesn't scream VALID ME

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (RemieRichards) [inline code/modules/uplink/uplink_item.dm]: traitors can buy syndi hardsuits, let them buy the webbing too, --metabreaking--

**Comment by Bawhoppen** [comment]
it looks so ungodly stupid I don't know why someone would support this atleast get a fucking new unique sprite instead of reusing one holy fuck

**Comment by lzimann** [comment]
Re-add this if you ever find a use instead of adding unused content lying around in the code.

**Comment by Bawhoppen** [comment]
it is used though as I said???????????????

**Comment by FantasticFwoosh** [comment]
Sprite your own or ask someone to, see my previous points i've made before in that literally NO ONE rushed to defend the military belt being removed and instead bitched about it looking ugly enough to possibly veto it.

**Comment by Bawhoppen** [comment]
fwoosh I really think it would be better for you to commenting because you seem to lack a grasp on most PRs

**Comment by Bawhoppen** [comment]
I feel bad but seriously what you said is completely incorrect in every way

**Comment by Bawhoppen** [comment]
Can I please get a legitimate reason why this PR is not being accepted? All it does is change the rigs to be a correct subtype of the military belt, so we can have the oldfashioned belt (which is used in ruins) as well as the chest rigging.

**Comment by lzimann** [comment]
Because I don't like clothing bloat.

**Comment by Bawhoppen** [comment]
Explain how it is clothing bloat when it serves an explicit purpose and does not add any new sprite to the code

**Pull request closed without merging**